src/Bookings.js

```diff
@@ -12,7 +13,8 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchButton />
```

Comment:
I would include the SearchButton component inside the Search component rather than Bookings, as it is part of the functionality of the Search component